.github/workflows/delivery-chocolatey.yml

```diff
@@ -4,25 +4,49 @@ on:
   release:
     types:
       - released
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: The release tag to distribute
+        required: true
 
 env:
   CHOCO_PATH: chocolatey
 
+defaults:
+  run:
+    shell: bash
+
 jobs:
-  prepare-chocolatey:
-    runs-on: ubuntu-latest
+  deliver-chocolatey:
+    runs-on: windows-latest
```

Comment:
Was this tested on Windows? IIRC, the default shell would be powershell in which case I'm not sure the `run` commands would work as expected.

PS: There is a way to change the default shell: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun